### PR TITLE
HOP ON THE MAGIC SCHOOLBUS MOTHERFUCKERS

### PR DESCRIPTION
**Pull request by Cheridan**
:cl: Cheridan
add: Added another space ruin.
/:cl:

xl0pyht

### CODE REVIEW

**Comment by Durkel** [comment]
When will the memes end?

File: code/game/objects/structures/bus.dm [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/obj/structure/bus/
```
Comment (Xhuis) [inline code/game/objects/structures/bus.dm]: Maybe make this a `fluff` structure?

**Comment by xxalpha** [comment]
needs a sombrero and poncho

**Comment by yackemflam** [comment]
THAT AINT A SHORT BUS

File: code/game/objects/structures/bus.dm (hunk 1 above)
Comment (Cheridan) [inline code/game/objects/structures/bus.dm]: ok

**Comment by Cheridan** [comment]
that's what makes it magic

**Comment by TechnoAlchemisto** [comment]
CAAAARLOSS

**Comment by Cheridan** [comment]
@yackemflam Thanks for the idea! I updated the map with an Adminbus.

**Comment by FlavoredCactus** [comment]
Needs MEME PR tag.

**Comment by ike709** [comment]
Is coderbus or adminbus the short one?

**Comment by RemieRichards** [comment]
@ike709 the smaller one is adminbus

**Comment by demonfiren** [comment]
Adminbus is obviously the one that doesn't have oranges.

**Comment by RemieRichards** [comment]
@demonfiren which is the smaller one, as I said.

**Pull request merged**